digest.go

```diff
@@ -214,10 +225,13 @@ func (da *DigestAuth) CheckAuth(r *http.Request) (username string, authinfo *str
 	return auth["username"], &info
 }
 
-// Default values for ClientCacheSize and ClientCacheTolerance for DigestAuth
+// Default values for ClientCacheSize, ClientCacheTolerance, ClientCachePurgeCount,
+// and ClientCacheTTL for DigestAuth.
 const (
-	DefaultClientCacheSize      = 1000
-	DefaultClientCacheTolerance = 100
+	DefaultClientCacheSize       = 1000
+	DefaultClientCacheTolerance  = 100
+	DefaultClientCachePurgeCount = 200
```

Comment:
Discussed offline. You're right, this would be easy to misconfigure especially now that a TTL was added. I've set the number of entries purged to the original `2*cache tolerance`.